xc/xenops_server_xen.ml

```diff
@@ -438,7 +437,7 @@ module Mem = struct
 		let (_: unit option) = wrap (fun () -> transfer_reservation_to_domain_exn dbg domid r) in
 		()
 
-        let query_reservation_of_domain dbg domid =
+				let query_reservation_of_domain dbg domid =
```

Comment:
It was a weird mixture of spaces and tabs. I will change it to all-tabs like the rest of the module.